### ISSUE TITLE
Set different paths for input and output data

### ISSUE THREAD

**Issue by reckoner165** [amoghpj/swar-synthesizer#8]
The current structure contains a single `data` directory that contains input .txt files as well as the output .wav. These should be moved to separate directories for clarity.

**Comment by onstash** [comment]
I agree with @reckoner165 on this. This enhancement also prevents accidental corruption of input files.

A possible solution is to have defaults for `input` and `output` directories but also make it configurable by having a way to provide `input` and `output` directories as environment variables (accessed via os.environ.get(...)` or arguments (utilizing [`argparse`](https://docs.python.org/3/library/argparse.html) - a utility for writing command line interfaces).
